Procfile.development

```diff
@@ -1,4 +1,3 @@
-web: bundle exec rails server -b 0.0.0.0
+web: bundle exec rails server -b 0.0.0.0 -p 3000
 worker: bundle exec sidekiq -C config/sidekiq.yml
-vite: bin/vite dev
-pusherfake: pusher-fake -i ${PUSHER_APP_ID:-123456} --socket-host 0.0.0.0 --socket-port ${PUSHER_WS_PORT:-45449} --web-host 0.0.0.0 --web-port ${PUSHER_PORT:-8888} -k ${PUSHER_APP_KEY:-centralakey} -s ${PUSHER_APP_SECRET:-centralsecret}
+pusherfake: PUSHER_FAKE=1 pusher-fake -i ${PUSHER_APP_ID:-123456} --socket-host 0.0.0.0 --socket-port ${PUSHER_WS_PORT:-45449} --web-host 0.0.0.0 --web-port ${PUSHER_PORT:-8888} -k ${PUSHER_APP_KEY:-centralakey} -s ${PUSHER_APP_SECRET:-centralsecret}
```

Comment:
There is an annoying issue with this process that looks to be a bug in Foreman. Foreman sends a SIGTERM to all child processes and then they are terminated. However, for pusher-fake specifically, the resources are not fully released by the process and it keeps the port 8888 busy. I couldn't figure out how to solve this, if you have any suggestion I'd appreciate it. For now, you can manually kill the process after exiting Foreman.